webviz_subsurface/plugins/_multiple_regression.py

```diff
@@ -0,0 +1,1129 @@
+                        clearable=False,
+                        value=self.ensembles[0],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div("Response:", style={"font-weight": "bold"}),
+                    dcc.Dropdown(
+                        id=self.uuid("responses"),
+                        options=[{"label": ens, "value": ens} for ens in self.responses],
+                        clearable=False,
+                        value=self.responses[0],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Parameters:",
+                        style={
+                            "font-weight": "bold",
+                            "display": "inline-block",
+                            "margin-right": "10px",
+                        },
+                    ),
+                    dcc.RadioItems(
+                        id=self.uuid("exclude-include"),
+                        options=[
+                            {"label": "Exclusive", "value": "exc"},
+                            {"label": "Subset", "value": "inc"},
+                        ],
+                        value="exc",
+                        labelStyle={"display": "inline-block"},
+                        style={"fontSize": ".80em"},
+                    ),
+                    dcc.Dropdown(
+                        id=self.uuid("parameter-list"),
+                        options=[{"label": ens, "value": ens} for ens in self.parameters],
+                        clearable=True,
+                        multi=True,
+                        placeholder="",
+                        value=[],
+                        style={"marginBottom": "20px"},
+                    ),
+                ]
+            ),
+            html.Div("Filters:", style={"font-weight": "bold"}),
+            html.Div(children=self.filter_layout),
+            html.Div(
+                [
+                    html.Div(
+                        "Model settings:", style={"font-weight": "bold", "marginTop": "20px"},
+                    ),
+                    html.Div(
+                        "Interaction", style={"display": "inline-block", "margin-right": "10px"},
+                    ),
+                    dcc.Slider(
+                        id=self.uuid("interaction"),
+                        min=0,
+                        max=2,
+                        step=None,
+                        marks={0: "Off", 1: "2 levels", 2: "3 levels"},
+                        value=0,
+                    ),
+                ]
+            ),
+            html.Div(
+                [
+                    html.Div(
+                        "Max number of parameters",
```

Comment:
Think "Max number of parameters" is confusing. Both because we include parameter interactions (which are not _parameters_), and because it is not a hard max (possible to go over if the last step is a step where an interaction is added). Should rephrase this